2_Operations/work-process.md

```diff
@@ -3,14 +3,21 @@
 
 ## Schedule
 
- * Every weekday morning we hold a daily standup meeting.
+ * Every **weekday** morning we hold a daily standup meeting:
+   * 10:00 CE(S)T for the whole company in the Niteo Zoom room
+   * followed immediately by separate standups for Easy Blog Networks and WooCart, in their own Zoom rooms
  * Every weekday Niteans should check their [GitHub notifications](https://github.com/notifications) so that they don't miss things that involve them.
 
 Sprint meetings replace the daily standup on these days:
 
- * A Wednesday is the start of a sprint with sprint planning.
- * The Tuesday, two weeks later, marks the end of the sprint with a sprint retrospective.
- * Then Wednesday, the next day, is usually the start of the next sprint.
+ * A **Wednesday** is the start of a sprint with sprint planning.
+   * 10:00 CE(S)T for the whole company in the Niteo Zoom room
+   * followed immediately by separate plannings for Easy Blog Networks and WooCart, in their own Zoom rooms
+ * The **Tuesday**, two weeks later, marks the end of the sprint with a sprint retrospective and company alignment meeting.
+   * 10:00 CE(S)T for Niteo
+   * followed immediately by retrospective for Easy Blog Networks 
+   * 11:00 CE(S)T for WooCart
```

Comment:
Can we do EBN & WooCart retros simultaneously?

Reply:
That would mean that there is no chance that we (or anyone else) can have an overview of both projects. Planning can be simultaneous, but I'd leave retros separate for now.